db/seeds.rb

```diff
@@ -18,6 +19,7 @@
                   major: Faker::Commerce.department,
                   current: gen > Time.new.year - 2009,
                   email: Faker::Internet.email,
+                  phone: Phonelib.parse(Faker::PhoneNumber.phone_number).sanitized,
```

Comment:
Let's take out the Faker here since it will probably error also. Try running `rake db:reset` to ensure the seed file works!